From 101c032478eaf775f31f2f074584438390286d41 Mon Sep 17 00:00:00 2001
From: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Date: Tue, 20 Nov 2018 14:21:45 +0200
Subject: [PATCH 5183/5242] MLKU-24-9 sound: soc: fix unused warning

commit  57706103cc17a64439682ab7cb98d27ef2a68bba from
https://source.codeaurora.org/external/imx/linux-imx.git

When CONFIG_PM is not defined, the runtime suspend and resume
are unused so we should use __maybe_unused attribute.

Signed-off-by: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
(cherry picked from commit b2285d1937872b5023074644d0a8f7c9a2e3158a)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_micfil.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/fsl/fsl_micfil.c b/sound/soc/fsl/fsl_micfil.c
index 81a60a1..fe821a9 100644
--- a/sound/soc/fsl/fsl_micfil.c
+++ b/sound/soc/fsl/fsl_micfil.c
@@ -2401,7 +2401,7 @@ static int __maybe_unused fsl_micfil_runtime_resume(struct device *dev)
 #endif /* CONFIG_PM*/
 
 #ifdef CONFIG_PM_SLEEP
-static int fsl_micfil_suspend(struct device *dev)
+static int __maybe_unused fsl_micfil_suspend(struct device *dev)
 {
 	struct fsl_micfil *micfil = dev_get_drvdata(dev);
 	int ret;
@@ -2421,7 +2421,7 @@ static int fsl_micfil_suspend(struct device *dev)
 	return 0;
 }
 
-static int fsl_micfil_resume(struct device *dev)
+static int __maybe_unused fsl_micfil_resume(struct device *dev)
 {
 	struct fsl_micfil *micfil = dev_get_drvdata(dev);
 	int ret;
-- 
1.7.9.5

